CHANGELOG.md

```diff
@@ -1,7 +1,8 @@
 ## v0.9.0-rc26 [unreleased]
 
 ### Bugfixes
-- [#2297](https://github.com/influxdb/influxdb/pull/2297) -- create /var/run during startup. Thanks @neonstalwart.
+- [#2297](https://github.com/influxdb/influxdb/pull/2297): create /var/run during startup. Thanks @neonstalwart.
+- [#2310](https://github.com/influxdb/influxdb/pull/2310): all socket ports used up
```

Comment:
This is actually PR #2312. Can I suggest the message should be something like "Broker should re-use HTTP client when invoking CQs"?